PWG/EMCAL/EMCALtasks/AliEmcalCorrectionCellEnergy.cxx

```diff
@@ -154,6 +158,19 @@ Int_t AliEmcalCorrectionCellEnergy::InitRecalib()
     
     contRF = std::unique_ptr<AliOADBContainer>(static_cast<AliOADBContainer *>(recalibFile->Get("AliEMCALRecalib")));
   }
+  else if (fCustomRecalibFilePath!="")
```

Comment:
For my education, why not just make `fBasePath` settable from the YAML? As far as I can tell from looking at this quickly, I think it can achieve the same thing as this PR. We just never made fBasePath settable because we didn't know of any interest

Reply:
fBasePath would mean that all OADB objects would need to be within this path. this seems unreasonable if one only wants to set a single OADB object. But, of course, you can also make a setter for fBasePath if you need

Reply:
`fBasePath` is a member function - it's not static. So it's set per component. I think every component accesses only one OADB object per component (except for the energy calib, which accesses energy and temperature - these probably should be split anyway). So perhaps it is necessary for the energy calibration, but I don't think we need it for the bad channel